assets/javascripts/discourse/templates/components/chat-live-pane.hbs

```diff
@@ -37,6 +37,8 @@
 
 {{chat-retention-reminder chatChannel=chatChannel}}
 
+<div class="chat-message-actions-mobile-anchor"></div>
```

Comment:
Longer term I want to move this here for real, but this requires many different refactoring first